composer.json

```diff
@@ -244,7 +244,10 @@
         ],
         "unit-tests": [
             "@php -d date.timezone=UTC ./vendor/phpunit/phpunit/phpunit -c tests/Unit/phpunit.xml"
-        ]
+        ],
+        "php-cs-fixer": "@php ./vendor/bin/php-cs-fixer fix --path-mode=intersection",
```

Comment:
Added php-cs-fixer (and phpstan as I'm at it) commands in the composer.json, so that we can easily launch it in the way that is ISO with the CI